src/main/java/group1/mips_simulator/components/Computer.java

```diff
@@ -2,12 +2,13 @@
 
 
 import group1.mips_simulator.components.cpuParts.CPU;
-import group1.mips_simulator.components.cpuParts.Register;
 import group1.mips_simulator.components.cpuParts.RomLoader;
+import group1.mips_simulator.components.instructionExecution.ExecutionResult;
+import group1.mips_simulator.components.instructionExecution.InstructionExecutions;
 import group1.mips_simulator.components.instructionParts.Field;
-import group1.mips_simulator.components.instructionParts.Instruction;
-import group1.mips_simulator.components.instructionParts.InstructionFactory;
-import group1.mips_simulator.components.instructionParts.RXIA_Instruction;
+import group1.mips_simulator.components.instructionParts.instruction.Instruction;
+import group1.mips_simulator.components.instructionParts.instruction.InstructionFactory;
+import group1.mips_simulator.components.instructionParts.instruction.RXIA_Instruction;
```

Comment:
trying to understand the directory structure change (not that im against organization lol) but why do we have an instruction folder inside instructionParts now?

I'll also make a recommendation for a minor refactor:
instead of `components.instructionParts.instruction.`... and a seperate `components.instructionExecution`
id like to suggest
`components.instruction.[parts|execution]` (like the ends of the directory would be part models, or execution implementations, maybe named iParts/iExecution or something, or keep the same names if we want to stay verbose;  but main point being these seem like things that should both be part of a single instruction package/module)

Thoughts?